react/components/Payment/ConnectorResponseInfo.tsx

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types'
+import React, { FunctionComponent } from 'react'
+import { injectIntl, InjectedIntlProps, intlShape } from 'react-intl'
+
+const ConnectorResponseInfo: FunctionComponent<Props & InjectedIntlProps> = ({
+  label,
+  value,
+  intl,
+}) => {
+  return (
+    <div className="more-info__data__item f7">
+      <span className="more-info__data__item__label mr2 fw5">
+        {intl.formatMessage({
+          id: `store/paymentData.connectorResponse.${label}`,
+          defaultMessage: label,
+        })}
+        :
+      </span>
+      <span className="more-info__data__item__value">{value}</span>
+    </div>
+  )
+}
+
+ConnectorResponseInfo.propTypes = {
```

Comment:
No need for the propTypes

Reply:
@dhirama with TypeScript, there is no need for PropTypes, as the type checks are made in compile time